File: Data Structures & Algorithms/two-integer-sum-ii/submission-3.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int pok1 = 0;
        while (pok1<numbers.length) {

            for (int k=pok1+1;k<numbers.length;k++) {
                if (numbers[pok1]+numbers[k]==target) {
                    return new int[]{pok1+1,k+1};
                }

            }
            pok1++;
        }
        return new int[]{};
    }
}

File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-3.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        kopija = nums
        levo = 0
        desno = len(nums)-1
        while levo<=desno:
            medijana = (levo+desno)//2
            if target==nums[medijana]:
                return kopija.index(nums[medijana])
            elif target in nums[levo:medijana]:
                nums = nums[levo:medijana]


            elif target in nums[medijana+1: desno+1]:
                nums = nums[medijana + 1:desno + 1]

            else:
                return -1
            levo = 0
            desno = len(nums)-1